common/source/docs/common-StampH743.rst

```diff
@@ -1,172 +1,266 @@
++----------------------+----------------------+----------------------+
+| Barometer            | BMP280               | BMP280               |
++----------------------+----------------------+----------------------+
+| Compass              | BMM150               | Not Fitted           |
++----------------------+----------------------+----------------------+
+| Micro SD Card        | Yes                  | Yes                  |
++----------------------+----------------------+----------------------+
+| **IO**               |                      |                      |
++----------------------+----------------------+----------------------+
+| PWM                  | 10                   | 10                   |
++----------------------+----------------------+----------------------+
+| CAN                  | 2                    | 2                    |
++----------------------+----------------------+----------------------+
+| UART                 | 8 (3 with flow       | 8 (None with flow    |
+|                      | control)             | control)             |
++----------------------+----------------------+----------------------+
+| I2C                  | 2                    | 2                    |
++----------------------+----------------------+----------------------+
+| External SPI         | 1 (With custom FW    | 1 (With custom FW    |
+|                      | build)               | build)               |
++----------------------+----------------------+----------------------+
+| Ethernet             | Yes                  | No                   |
++----------------------+----------------------+----------------------+
+| Safety Button        | Yes                  | Yes                  |
++----------------------+----------------------+----------------------+
+| Analog               | 2                    | 2                    |
++----------------------+----------------------+----------------------+
+| Buzzer (PWM 11)      | Yes                  | Yes                  |
++----------------------+----------------------+----------------------+
+| USB                  | Yes                  | Yes                  |
++----------------------+----------------------+----------------------+
+| **Power**            |                      |                      |
++----------------------+----------------------+----------------------+
+| Input Voltage        | 5v                   | 5v                   |
++----------------------+----------------------+----------------------+
+| Typical Current      | 0.4 A                | 0.4 A                |
+| Consumption          |                      |                      |
++----------------------+----------------------+----------------------+
+| Independent Power    | 6 Separate Power     | No                   |
+| Domains              | Domains              |                      |
++----------------------+----------------------+----------------------+
+
+
+
+UART Mapping
```

Comment:
this needs to be in the same format as previous including the default protocols and dma capability